docs/source/notes/tensorboard_logger.md

```diff
@@ -63,106 +63,6 @@ trainer.fit()
 Now, run this code and if all goes well, your loss and metric results will be logged Tensorboard log files, which will be written to "./my_tensorboard_logs".
 See [these instructions](#viewing-your-results-locally) for viewing your results in the Tensorboard viewer.
 
-## Logging to Tensorboard using YAML configs
```

Comment:
Maybe remove "pythonically" from the above section now that there is no yaml section